input/schemas/paper_source_info.schema.yaml

```diff
@@ -19,28 +19,28 @@ properties:
       ra:
         type: object
         additionalProperties: false
-        description: Right Ascension
+        description: Right Ascension in deg
```

Comment:
I don't remember actually what I do with these values, and if they have to be in deg.
Looks like I put hms and dms strings elsewhere:
```
$ ack dec input/papers
input/papers/2007/2007A%26A...469L...1A/tev-000030.yaml
9:  dec: {val: 5d48m20s, err: 28 arcmin}

input/papers/2012/2012A%26A...545L...2H/tev-000027.yaml
10:  dec: {val: -69d09m57s, err: 11 arcmin}

input/papers/2012/2012A%26A...548A..38A/tev-000037.yaml
10:  dec: {val: -45d36m00s, err: 120}
17:  pa: {frame: radec, val: 41.0, err: 7.0}
```

The `ack` tool is nice to search for stuff: http://beyondgrep.com/

Ah, now I see. The type is `object` which means you should put the info into an "object" (called dict in Python):
```
ra: {val: XXX}
```
as in the other example files.